src/components/molecules/Schedule/Schedule.utils.ts

```diff
@@ -0,0 +1,16 @@
+import { getSecondsFromStartOfDay, ONE_HOUR_IN_SECONDS } from "utils/time";
+
+export const HOUR_WIDTH = 200; // px
```

Comment:
@0xdevalias 

I've seen this a few times by now. I wonder if we should include the units into the const name. It sounds more logical 🤷

Reply:
Also, should be moved to settings & have a more descriptive name

Reply:
I’m ok with having it in the car james so long as they don’t make it overly long/annoying. Eg. ‘_PX’ for pixels, ‘_MS’ for milliseconds, ‘_SEC’ for seconds.

I really strongly dislike the massively long constant names we have in ‘utils/time’ currently.

Reply:
renamed https://github.com/sparkletown/sparkle/pull/1302/commits/3773b544eaa9b24083615b54b444167ca724eff2
but haven't moved to settings yet.